core/build.sbt

```diff
@@ -1,5 +1,26 @@
 
-libraryDependencies ++= Seq(
-  "org.scalaz" %% "scalaz-core"       % "7.1.2",
-  "org.scalaz" %% "scalaz-concurrent" % "7.1.2"
-)
+val scalazVersion = sys.env.getOrElse("SCALAZ_VERSION", "7.1.11")
+val scalazMajorVersion = scalazVersion.take(3)
+
+libraryDependencies ++= {
+
+  Seq(
+    "org.scalaz" %% "scalaz-core"       % scalazVersion,
+    "org.scalaz" %% "scalaz-concurrent" % scalazVersion
+  )
+}
+
+// Add scalaz version to project version
+version := {
```

Comment:
@djspiewak can you take a look at this? I have been thinking about doing something similar, but want a 2nd opinion. I dislike the `a` convention and would prefer something else.